src/ontology/mondo-edit.obo

```diff
@@ -574805,6 +574808,17 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-12T13:35:21Z" xsd:
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7493" xsd:anyURI
 
+[Term]
+id: MONDO:1040012
+name: PI4KA-Related Disorder
```

Comment:
"related" and "disorder" should not be capitalized

Reply:
Comment addressed.